requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
 # Development specific requirements
 pytest==3.6.3
-pylint==2.0.1
-yapf==0.22.0
+pylint
+yapf
```

Comment:
I think we should have fixed versions, its always a pain in the ass if we end up having different versions of software and we end up with different bugs or something because we have a different setup. Given that this goes in the person's `venv` then I think it should def be version specific

Reply:
You say in the PR you removed the versions because they were too old, but I would suggest you just pin them back to something newer

Reply:
ok, but it will go outdated at some point and break

Reply:
done